src/mscorlib/src/System/IO/Path.cs

```diff
@@ -102,7 +102,12 @@ public static class Path
         // Make this public sometime.
         // The max total path is 260, and the max individual component length is 255. 
         // For example, D:\<256 char file name> isn't legal, even though it's under 260 chars.
+#if !PLATFORM_UNIX
```

Comment:
I think that double defining it is fine.

There are thousands of constant, structures and PInvoke definitions double defined between C# and C++. One more won't make a difference.